public/js/host.js

```diff
@@ -93,69 +154,83 @@
       startStopButton.classList.add('active');
       startStopButton.innerHTML = 'End Broadcast';
       document.getElementById('urlContainer').classList.remove('hidden');
-      displayUrl.innerHTML = playerUrl;
-      displayUrl.setAttribute('value', playerUrl);
       if (broadcast.rtmp) {
         rtmpActive.classList.remove('hidden');
       }
     } else {
       startStopButton.classList.remove('active');
-      startStopButton.innerHTML = 'Broadcast Over';
-      startStopButton.disabled = true;
+      startStopButton.innerHTML = 'Start Broadcast';
       rtmpActive.classList.add('hidden');
+      document.getElementById('rtmpLabel').classList.remove('hidden');
+      document.getElementById('rtmp-options').classList.remove('hidden');
     }
 
     signal(session, broadcast.status);
   };
 
   // Let the user know that the url has been copied to the clipboard
-  const showCopiedNotice = function () {
-    const notice = document.getElementById('copyNotice');
-    notice.classList.remove('opacity-0');
-    setTimeout(function () {
-      notice.classList.add('opacity-0');
-    }, 1500);
-  };
 
   const validRtmp = function () {
     const server = document.getElementById('rtmpServer');
     const stream = document.getElementById('rtmpStream');
 
     const serverDefined = !!server.value;
     const streamDefined = !!stream.value;
-    const invalidServerMessage = 'The RTMP server url is invalid. Please update the value and try again.';
-    const invalidStreamMessage = 'The RTMP stream name must be defined. Please update the value and try again.';
+    const invalidServerMessage =
+      'The RTMP server url is invalid. Please update the value and try again.';
+    const invalidStreamMessage =
+      'The RTMP stream name must be defined. Please update the value and try again.';
 
     if (serverDefined && !server.checkValidity()) {
       document.getElementById('rtmpLabel').classList.add('hidden');
+      document.getElementById('rtmp-options').classList.add('hidden');
       document.getElementById('rtmpError').innerHTML = invalidServerMessage;
       document.getElementById('rtmpError').classList.remove('hidden');
       return null;
     }
 
     if (serverDefined && !streamDefined) {
       document.getElementById('rtmpLabel').classList.add('hidden');
+      document.getElementById('rtmp-options').classList.add('hidden');
       document.getElementById('rtmpError').innerHTML = invalidStreamMessage;
       document.getElementById('rtmpError').classList.remove('hidden');
       return null;
     }
 
     document.getElementById('rtmpLabel').classList.remove('hidden');
+    document.getElementById('rtmp-options').classList.remove('hidden');
     document.getElementById('rtmpError').classList.add('hidden');
-    return { serverUrl: server.value, streamName: stream.value };
+    return { id: 'testId', serverUrl: server.value, streamName: stream.value };
```

Comment:
why is `testId` needed?

Reply:
I think it's because to create an RTMP you need to pass and Id https://tokbox.com/developer/rest/#start_broadcast @enricop89 to confirm

Reply:
I've replaced testId with a randomly generated Id

Reply:
Right, so it's not actually mandatory as per the docs. But since the new version of the sample app supports multiple simultaneous broadcast I think it makes sense to have an ID, otherwise the broadcast creation could fail as the name needs to be unique. The former version of the app supported only one broadcast at a time

>  For each RTMP stream, specify serverUrl (the RTMP server URL), streamName (the stream name, such as the YouTube Live stream name or the Facebook stream key), and (optionally) id (a unique ID for the stream). If you specify an ID, it will be included in the REST call response and the [REST method for getting information about a live streaming broadcast](https://tokbox.com/developer/rest/#get_info_broadcast).